google/cloud/compute/v1/compute.proto

```diff
@@ -582,7 +576,7 @@ message AllocationSpecificSKUReservation {
 
 }
 
-// Represents a customer-supplied encryption key
+//
```

Comment:
This is their change, not ours? (I'm surprised they deleted the comment)

Reply:
There are no any manual changes in the generated proto, so must be their change. They definitely do all sorts of things with their comments (I saw many edits before, though never seen a complete removal).